core/storage/user/gae_models.py

```diff
@@ -244,7 +284,8 @@ def is_normalized_username_taken(cls, normalized_username):
 
     @classmethod
     def get_by_gae_id(cls, gae_id):
-        """Returns a user model with given GAE user ID.
+        """Returns a user model with given GAE user ID. Will be deprecated
```

Comment:
No, please don't mark fields as "will be deprecated". Handle the deprecation in the deprecation PR.